framework/src/test/java/org/tron/core/services/http/GetRewardServletTest.java

```diff
@@ -0,0 +1,127 @@
+            new String[]{
+                "--output-directory", dbPath,
+            }, Constant.TEST_CONF
+    );
+  }
+
+  public MockHttpServletRequest createRequest(String contentType) {
+    MockHttpServletRequest request = new MockHttpServletRequest();
+    request.setMethod("POST");
+    request.setContentType(contentType);
+    request.setCharacterEncoding("UTF-8");
+    return request;
+  }
+
+  public void init() {
+    manager.getDynamicPropertiesStore().saveChangeDelegation(1);
+    byte[] sr = decodeFromBase58Check("27VZHn9PFZwNh7o2EporxmLkpe157iWZVkh");
+    delegationStore.setBrokerage(0, sr, 10);
+    delegationStore.setWitnessVote(0, sr, 100000000);
+  }
+
+  @Test
+  public void getRewardValueByJsonTest() {
+    init();
+    int expect = 138181;
+    String jsonParam = "{\"address\": \"27VZHn9PFZwNh7o2EporxmLkpe157iWZVkh\"}";
+    MockHttpServletRequest request = createRequest("application/json");
+    MockHttpServletResponse response = new MockHttpServletResponse();
+    request.setContent(jsonParam.getBytes());
+    try {
+      getRewardServlet.doPost(request, response);
+      String contentAsString = response.getContentAsString();
+      JSONObject result = JSONObject.parseObject(contentAsString);
+      int reward = (int)result.get("reward");
+      Assert.assertEquals(expect, reward);
+    } catch (UnsupportedEncodingException e) {
+      Assert.fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void getRewardValueTest() {
+    init();
+    int expect = 138181;
+    MockHttpServletRequest request = createRequest("application/x-www-form-urlencoded");
+    MockHttpServletResponse response = new MockHttpServletResponse();
+    mortgageService.payStandbyWitness();
+    request.addParameter("address", "27VZHn9PFZwNh7o2EporxmLkpe157iWZVkh");
+    getRewardServlet.doPost(request, response);
+    try {
+      String contentAsString = response.getContentAsString();
+      JSONObject result = JSONObject.parseObject(contentAsString);
+      int reward = (int)result.get("reward");
+      Assert.assertEquals(expect, reward);
+    } catch (UnsupportedEncodingException e) {
+      Assert.fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void getByBlankParamTest() {
+    MockHttpServletRequest request = createRequest("application/x-www-form-urlencoded");
+    MockHttpServletResponse response = new MockHttpServletResponse();
+    request.addParameter("address", "");
+    GetRewardServlet getRewardServlet = new GetRewardServlet();
+    getRewardServlet.doPost(request, response);
+    try {
+      String contentAsString = response.getContentAsString();
+      JSONObject result = JSONObject.parseObject(contentAsString);
+      int reward = (int)result.get("reward");
+      Assert.assertEquals(0, reward);
+      String content = (String) result.get("Error");
+      Assert.assertNull(content);
+    } catch (UnsupportedEncodingException e) {
+      Assert.fail(e.getMessage());
+    }
+  }
+
+  @After
+  public void deleteDatabase() {
```

Comment:
`BaseTest` has already done the release.

Reply:
Yes, it can be dispensed with here